package-lock.json

```diff
@@ -9100,6 +9114,13 @@
       "integrity": "sha1-ZUEYTMkK7qbG57NeJlkIJEPGYZg=",
       "requires": {
         "hoek": "2.x.x"
+      },
+      "dependencies": {
+        "hoek": {
+          "version": "2.16.3",
```

Comment:
This version is vulnerable.

Reply:
It is done by `npm` I have checked it by making a commit in my own repository and the message was not coming now.
![image](https://user-images.githubusercontent.com/25537606/42146282-6852847e-7de4-11e8-87ce-57da6c48b7c5.png)

Reply:
@praveenojha33 Please see: https://nodesecurity.io/advisories/566. And the message is the alert generated by Github's CVV report. I am not sure why is it not showing alert FYI, as npm suggested. Could you please once check in the dependency settings of the above project, if there seems any issue?